app/messages/_layout.email.html.erb

```diff
@@ -1,62 +1,116 @@
+<!DOCTYPE html>
 <html>
-<body style="background-color: #FFFFFF;">
+<head>
+  <meta name="viewport" content="width=device-width">
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>Canvas Email</title>
```

Comment:
Let's leave the title out for now, and just focus this PR on the responsiveness.  Thanks!